SimpleCalculator/SimpleCalculatorTest/Test.cs

```diff
@@ -0,0 +1,86 @@
+using System.Reflection;
+using SimpleCalculator.Business;
+using SimpleCalculator.Business.Enums;
+using System;
+
+namespace SimpleCalculatorTest;
+
+public class Test
+{
+    [Theory]
+    [InlineData(2, 3)]
+    [InlineData(5, 6)]
+    [InlineData(13, 6)]
+    public void Calculate_Sum_SumOfTwoNumbers(int firstOperand, int secondOperand)
+    {
+        int expected = firstOperand + secondOperand;
+        Calculator calculator = new Calculator();
```

Comment:
Expected value should be included in the InlineData defined above, and shouldn't be calculated again (here, it doesn't pose a problem, but in bigger projects you have to implement the method you are testing again just to calculate the expected value).